recipes/astyle/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+cmake -S . -B build \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_VERBOSE_MAKEFILE=ON \
+    -Wno-dev \
+    -DBUILD_TESTING=OFF \
+    ${CMAKE_ARGS}
+cmake --build build -j${CPU_COUNT}
+cmake --install build
```

Comment:
Is there a `make check` or `ctest` that we can run?

Reply:
Unfortunately there is no testing functionality implemented in the `CMakeLists.txt` for this package.